rules-reviewed/camel3/camel2/xml-renamed-components.windup.xml

```diff
@@ -79,5 +79,185 @@
+                    <link title="Camel 3 - Renamed Components" href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_renamed_components" />
+                    <quickfix type="REPLACE" name="camel-hdfs2-replace">
+                        <replacement>camel-hdfs2</replacement>
+                        <search>camel-hdfs</search>
+                    </quickfix>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="xml-renamed-components-00005">
+            <when>
+                <project>
+                    <artifact groupId="org.apache.camel" artifactId="camel-hdfs" />
+                </project>
+            </when>
+            <perform>
+                <hint title="`org.apache.camel:camel-hdfs` artifact has been replaced with the old `org.apache.camel:camel-hdfs2` artifact" effort="1" category-id="mandatory" >
+                    <message>`org.apache.camel:camel-hdfs` artifact has been replaced with the old `org.apache.camel:camel-hdfs2` artifact. The component now depends on Hadoop 2.x, upgrade efforts may vary.</message>
+                    <link title="Camel 3 - Renamed Components" href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_renamed_components" />
+                </hint>
+            </perform>
+        </rule>
+        <rule id="xml-renamed-components-00006">
+            <when>
+                <project>
+                    <artifact groupId="org.apache.camel" artifactId="camel-mina2" />
+                </project>
+            </when>
+            <perform>
+                <hint title="`org.apache.camel:camel-mina2` artifact has been renamed to `org.apache.camel:camel-mina`" effort="1" category-id="mandatory" >
+                    <message>`org.apache.camel:camel-mina2` artifact has been renamed to `org.apache.camel:camel-mina`. It’s corresponding component package was renamed from `org.apache.camel.component.mina2` to `org.apache.camel.component.mina`. The supported scheme is now `mina`.</message>
+                    <link title="Camel 3 - Renamed Components" href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_renamed_components" />
+                    <quickfix type="REPLACE" name="camel-mina2-replace">
+                        <replacement>camel-mina2</replacement>
+                        <search>camel-mina</search>
+                    </quickfix>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="xml-renamed-components-00007">
+            <when>
+                <project>
+                    <artifact groupId="org.apache.camel" artifactId="camel-mina" />
+                </project>
+            </when>
+            <perform>
+                <hint title="`org.apache.camel:camel-mina` artifact has been replaced with the old `org.apache.camel:camel-mina2` artifact" effort="2" category-id="mandatory" >
```

Comment:
[suggested efforts available](https://access.redhat.com/documentation/en-us/red_hat_application_migration_toolkit/4.3/html-single/rules_development_guide/index#how_story_points_are_estimated_in_rules) are `1` or `3` and i think `1` is fine 
```suggestion
                <hint title="`org.apache.camel:camel-mina` artifact has been replaced with the old `org.apache.camel:camel-mina2` artifact" effort="1" category-id="mandatory" >
```